recipe/meta.yaml

```diff
@@ -56,13 +55,14 @@ about:
   home: https://www.sqlite.org/
   license: Unlicense
```

Comment:
I believe license should be `blessing` and `license_url` should be removed in favor of `license_file` with a value of `LICENSE.md`. The license can be found here: https://sqlite.org/src/file?name=LICENSE.md&ci=trunk

Reply:
I left unlicense which is more explicit imho, and added LICENSE.md.